input/include/NovelRT/Input/Glfw/GlfwInputDevice.hpp

```diff
@@ -4,24 +4,28 @@
 #ifndef NOVELRT_INPUT_GLFW_GLFWINPUTDEVICE_H
 #define NOVELRT_INPUT_GLFW_GLFWINPUTDEVICE_H
 
-#ifndef NOVELRT_INPUT_GLFW_H
-#error NovelRT does not support including types explicitly by default. Please include Input.Glfw.h instead for the Input::Glfw namespace subset.
-#endif
-
```

Comment:
If we are still relying on a centralised header as per your last comment, we should probably revert this

Reply:
Resolved - i have banished the centralised header to the shadow realm.